test/format/format.sh

```diff
@@ -74,12 +75,16 @@ parallel_jobs=8
 smoke_test=0
 total_jobs=0
 verbose=0
+format_binary_cmd="./t"
```

Comment:
Don't really need `format_binary_cmd`, just use `format_binary`?

Reply:
Yes, I have removed format_binary_cmd and using just format_binary instead. Thanks.